modules/avatar/react/stories/stories.tsx

```diff
@@ -13,7 +13,7 @@ const handleAvatarButtonClick = (e: React.SyntheticEvent) => {
   console.log('AvatarButton clicked');
 };
 
-storiesOf('Avatar/Default', module)
+storiesOf('Tokens|Avatar/Default', module)
```

Comment:
Are we sure this should be in tokens? I would call this a component.

Reply:
@anicholls Ok, I'm ok with moving these into `Components` I think especially buttons would make sense...We may likely revisit this down the road but I think it's ok for now.

Reply:
Okay, going to ping and see if he has any thoughts...Not really sure where we would want to put it in the categories that we have currently.

Reply:
Hey everyone, this is an interesting one, if we see, both of them, 'Avatar' and 'Avatar Button' are basically the same from a design pov, the specs for both will be the same since there are no states applied to this button. The only difference is that one is treated as a button since a link is added while the other one's primary function is just to display the avatar. If it makes sense for you we can have 'Avatar Button' under 'Buttons'. Just need to update the taxonomy.

Reply:
I still think `Avatar` and `AvatarButton` should be grouped together, I just think "Avatar" could be its own section under components.